stdlib/3/tokenize.pyi

```diff
@@ -3,9 +3,10 @@ from builtins import open as _builtin_open
 import sys
 from token import *  # noqa: F403
 
-COMMENT: int
-NL: int
-ENCODING: int
+if sys.version_info < (3, 7):
```

Comment:
Since they are still exposed by tokenize at runtime, shouldn't we also keep them here even in 3.7 and higher? The current code would make it hard for people to write type checked code using these that runs in both 3.6 and 3.7.

Reply:
This should be handled by `from token import *`, I think.

Reply:
You're right, I missed that